workflow/rocoto/gfs_tasks.py

```diff
@@ -613,7 +621,7 @@ def snowanl(self):
     def esnowrecen(self):
 
         deps = []
-        dep_dict = {'type': 'task', 'name': f'{self.run.replace("enkf","")}_prepsnowobs'}
+        dep_dict = {'type': 'task', 'name': f'{self.run.replace("enkf","")}prepsnowcover'}
```

Comment:
We want the underscore in the job name here.
```suggestion
        dep_dict = {'type': 'task', 'name': f'{self.run.replace("enkf","")}_prepsnowcover'}
```

Reply:
Thanks @KateFriedman-NOAA for catching this, and I made the changes as suggested.